### DISCUSSION TITLE
Short description for README.md missing?

### DISCUSSION THREAD

**Discussion by alexbergsland** [outdoorbits/little-backup-box#165] [Ideas]
Am I blind or is the short description of what this project actually does missing from the [README.md](https://github.com/outdoorbits/little-backup-box/blob/main/README.md) ? :)

I stumbled across this project and am curious of what it does, a short description should be at the top of the readme I think.
I am guessing that you send backups to this box through the network but it's not clear.

**Comment by outdoorbits** [comment]
What's about the first sentence:

Software that turns a single-board computer into a versatile and pocketable backup appliance.

**Comment by alexbergsland** [comment]
Hehe ok, sorry. I missed that, I am used to seeing it in the readme, my bad.
Thank you.
